TF_Service_dotNet/TouchFreeTests/PositionTrackers/IndexTipTrackerTests.cs

```diff
@@ -13,12 +13,17 @@ public IndexTipTrackerTests()
             sut = new IndexTipTracker();
         }
 
-        [TestCase(1000, 2000, 3000, 1, 2, 3)]
-        [TestCase(500, 1000, 2000, 0.5f, 1, 2)]
-        public void CalculatePositions_ValidHandPosition_Returns2dVector(float fingerTipX, float fingerTipY, float fingerTipZ, float cursorX, float cursorY, float distanceFromScreen)
+        private static object[] positionsData = new object[]
+        {
+            new object[] { new Vector3(1000, 2000, 3000), new Vector2(1, 2), 3 },
+            new object[] { new Vector3(500, 1000, 2000), new Vector2(0.5f, 1), 2 }
+        };
+
+        [TestCaseSource(nameof(positionsData))]
+        public void CalculatePositions_ValidHandPosition_Returns2dVector(Vector3 wristPositionMm, Vector2 expectedPositionM, float distanceFromScreenM)
```

Comment:
Not overly keen on the formatting of this, but I don't know of a better way

I think the readability of this is better than before though so I am happy with it :) (particularly as parameters)